src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -921,10 +922,10 @@ export default {
 
             // redirect to Edit web app to correct this Incorporation Application
             const editUrl = sessionStorage.getItem('EDIT_URL')
-            const url = `${editUrl}${this.getEntityIncNo}/correction?correction-id=${draftCorrectionId}`
-            // assume Edit URL is always reachable
-            window.location.assign(url)
+            const correctionUrl = `${editUrl}${this.getEntityIncNo}/correction?correction-id=${draftCorrectionId}`
+            this.redirect(correctionUrl)
```

Comment:
This feels messy: I'm asking for the spinner twice (line 911 and inside `redirect()`).

Reply:
Obsolete.